pokemon-showdown

```diff
@@ -16,15 +18,18 @@ try {
 try {
 	require.resolve('sockjs');
 } catch (e) {
-	let command = 'npm install --production';
-	console.log('Installing dependencies: `' + command + '`...');
-	require('child_process').spawnSync('sh', ['-c', command], {stdio: 'inherit'});
+	console.log('Installing dependencies...');
+	spawnSync('npm', ['install', '--production'], {stdio: 'inherit', shell: true});
```

Comment:
So I wouldn't need to spawn sh -c or cmd /C /S depending on OS separately, since that's all handled internally anyways when spawning with that enabled. Whether or not that's actually needed though, I can check tomorrow